src/Statistics.cpp

```diff
@@ -306,6 +388,14 @@ format_human_readable(const Config& config)
       continue;
     }
 
+    // don't show cache directory info if reading from a log
+    if (counters.from_log()
+        && (statistic == Statistic::cleanups_performed
+            || statistic == Statistic::files_in_cache
+            || statistic == Statistic::cache_size_kibibyte)) {
```

Comment:
I suggest adding a flag in `k_statistics_counters` for this instead. Maybe named `FLAG_NOSTATSLOG`?